src/sql/workbench/contrib/executionPlan/browser/media/executionPlan.css

```diff
@@ -33,7 +33,7 @@
 The actual sash element constructed by code. Important is used here because the width of the sash is fixed.
 However we always want it to be the width of the container it is resizing.
 */
-.eps-container .execution-plan-sash > div {
+.eps-container .execution-plan-sash>div {
```

Comment:
nit : But generally it's best to leave the spaces. Makes it easier to read and parse the selector

Reply:
It seems that auto format got rid of it

Reply:
Huh, annoying. Ok well if the formatter is doing it then fine to leave it like that then.